Externals.cfg

```diff
@@ -62,5 +62,14 @@ hash = 0a86f73
 local_path = sorc/AQM-utils
 required = True
 
+[uwtools]
+protocol = git
+repo_url = https://github.com/ufs-community/workflow-tools
+# Specify either a branch name or a hash but not both.
+# branch = develop
+hash = e1b3b6f
+local_path = ush/python_utils/uwtools
```

Comment:
I do have to say it is a little confusing that the `uwtools` directory isn't under `sorc` directory, but if you check Externals.cfg, you'll see where it is downloaded. Given that the rest of the external packages are used within the model and `uwtools` is a python package for the workflow, I have no problem placing the directory under `ush/python_utils/uwtools`.